.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -7,14 +7,14 @@ body:
     label: XOOPS Version
     description: Provide the XOOPS version that you are using.
-    options: [ '2.5.11', '2.5.12' ]
+    options: [ '2.5.11', '2.5.12', '2.5.13' ]
   validations:
     required: true
 - type: dropdown
   attributes:
     label: PHP Version
     description: Provide the PHP version that you are using.
-    options: [ '7.4', '8.0','8.1','8.2','8.3', '8.4' ]
+    options: [ '7.4', '8.0', '8.1', '8.2', '8.3', '8.4', '8.5' ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)